idd/Energy+.idd.in

```diff
@@ -49467,13 +49467,106 @@ Coil:Cooling:Water:DetailedGeometry,
   A7 , \field Condensate Collection Water Storage Tank Name
        \type object-list
        \object-list WaterStorageTankNames
-  N17; \field Design Water Temperature Difference
+  N17, \field Design Water Temperature Difference
        \type real
        \units deltaC
        \minimum> 0.0
        \note This input field is optional. If specified, it is used for sizing the Design Water Flow Rate.
        \note If blank or omitted, the Loop Design Temperature Difference value specified in Sizing:Plant
        \note object is used for sizing the Design Water Flow Rate.
+  N18; \field Design Water Inlet Temperature
+       \type real
+       \units C
+       \minimum> 0.0
+       \note This input field is optional. If specified, it is used for sizing the coil Design Geomtery
+       \note Parameters. If blank or omitted, the Design Loop Exit Temperature value specified in 
+       \note Sizing:Plant object is used for sizing the coil Design Geomtery Parameters.
+
+CoilSystem:Cooling:Water,
+       \memo Virtual container component that consists of a water cooling coil
+       \memo and its associated controls. This control object supports the
+       \memo available water coil types and may be placed directly on an
+       \memo air loop branch or in an outdoor air equipment list.
+       \min-fields 6
+   A1, \field Name
+       \required-field
+       \reference WaterCoilSystemName
```

Comment:
This \reference name isn't used anywhere, so it's not needed. I see that Coilsystem:Cooling:DX has a similar nade that is referenced in FaultModel:TemperatureSensorOffset:CoilSupplyAir. I'm not suggesting this PR needs to add fault support for this system type, but you should post a new issue to have it added at some point.

Reply:
Removed *\reference WaterCoilSystemName*.

Reply:
@mjwitte If this field is autosizable, the auto-sized value is set to the condenser loop Design Loop Exit Temperature value and usually it ranges from 25C - 29C. This values is greater than the design supply air temperature, which leads to sizing problem. This field was added to overcome this limitation.

Reply:
True, for your file, leaving it blank or setting it to autosize won't make sense. But when this coil is used for other applications, when it's blank or autosize (as suggested) it would grab the connected loop value. The behavior doesn't change here, it's a matter of do we want blank to be significant or do we want to say "autosize" to be consistent with other similar fields.  And by that argument "Design Water Temperature Difference" should probably be autosizable also with a default of autosize. 

@rraustad @Myoldmopar Any opinions on blank vs autosize for fields like this?

Reply:
And if we agree to \default autosize, min-fields will need to be increased to include this field so that the default gets applied.

Reply:
> 
> 
> This \reference name isn't used anywhere, so it's not needed. I see that Coilsystem:Cooling:DX has a similar nade that is referenced in FaultModel:TemperatureSensorOffset:CoilSupplyAir. I'm not suggesting this PR needs to add fault support for this system type, but you should post a new issue to have it added at some point.

Added new issue #8941